### ISSUE TITLE
My terminal is all blue after exiting vim

### ISSUE THREAD

**Issue by ivanistheone** [aam1r/sachet#16]
```
set nocompatible

" Pathogen
call pathogen#infect()
call pathogen#helptags()

set statusline=%<\ %n:%f\ %m%r%y%=%-35.(line:\ %l\ of\ %L,\ col:\ %c%V\ (%P)%)
filetype plugin indent on

syntax on
set number
set mouse=a
set mousehide

set spell
set hlsearch
set showmatch
set autoindent
set history=1000
set cursorline
set expandtab
set shiftwidth=2
set tabstop=2
set softtabstop=2

" Nerdtree
autocmd vimenter * NERDTree
let NERDTreeShowBookmarks=1
let NERDTreeChDirMode=0
let NERDTreeQuitOnOpen=0
let NERDTreeMouseMode=2
let NERDTreeShowHidden=1
let NERDTreeIgnore=['\.pyc','\~$','\.swo$','\.swp$','\.git','\.hg','\.svn','\.bzr']
let NERDTreeKeepTreeInNewTab=1
let g:nerdtree_tabs_open_on_gui_startup=0


set background=dark
colorscheme solarized
```

**Comment by ivanistheone** [comment]
I forgot to say -- i use iTerm Build 0.9.6

![Screen shot 2013-03-12 at 5 30 30 PM](https://f.cloud.github.com/assets/163966/251060/28001920-8b5c-11e2-85b0-d8387cb97e7f.png)

**Comment by ivanistheone** [comment]
The problem with the texrminal being garbled might have been due to the old version of iTerm I was using.
I switched to a newer version from here  http://www.iterm2.com/#/section/home and things seem to work now.

These are the relavant lines in my .vimrc:

   set cursorline
   set background=dark
   colorscheme Tomorrow-Night
   let &t_Co=256

I also went into Profiles -> Open Profiles -> Edit profiles -> Terminal tab -> set Report Term type = xterm-256color.

Between these two changes, terminal works nice now.
